sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/administration/models/EvaluationsCreateHeaders.java

```diff
@@ -16,6 +17,16 @@ public final class EvaluationsCreateHeaders {
     @JsonProperty(value = "Location")
```

Comment:
Should this be in Impl class?
Public class with setter?

Reply:
Please refer to line 37 in PersonalizerCustomization class regarding why this class is public. It should not be but we are making an exception in this case. I had discussed this with Srikanta and he was ok with it since it is in beta.